src/components/footer/footer.styles.js

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components';
+import { NavLink } from 'react-router-dom';
+import { ReactComponent as Logo } from '../../assets/logo.svg';
+
+export const NavContainer = styled.nav`
+  height: 125px;
+  margin-left: 250px;
+  margin-right: 250px;
```

Comment:
The problem with using fixed margins here is that the app is not responsive. This is what the footer looks like on small screens:

![image](https://user-images.githubusercontent.com/9722792/120156704-84d91e00-c1f2-11eb-80fc-9ea56b152ff8.png)

The fixed margin is kept even though there isn't enough space (e.g. on screens with width < 500px).

Alternatively, you can use `width: 100%` together with a `max-width` according to the designs. That way the footer will take the complete screen width by default but never exceed the width given in the designs on large screen.